### ISSUE TITLE
Update methods simpler overloads

### ISSUE THREAD

**Issue by sylvlecl** [powsybl/pypowsybl#238]
* **Do you want to request a *feature* or report a *bug*?**

Feature.

* **What is the current behavior?**

Update methods only accept `pandas.DataFrame` as input, which can be a little cumbersome 
for simple use cases, for example when you only want to update one value. For example,
a minimalist use case will look like :  
```python
network.update_generators(pd.DataFrame(index=['GEN'], data={'target_p': [ 350 ] }))
```

* **What is the expected behavior?**

We could provide simpler overloads which would not take a dataframe as input, but a list of named arguments:
```python
network.update_generators(id='GEN', target_p=350)
network.update_generators(id=['GEN1', 'GEN2'], target_p=[350, 400])
network.update_generators(id=['GEN1', 'GEN2'], target_p=some_numpy_array)
```

* **What is the motivation / use case for changing the behavior?**

Ease of use, in particular for simple use cases.

**Comment by sylvlecl** [comment]
Note:
to implement this, we should use keyword arguments, and transform keywords arguments in dataframes if no dataframe is provided as the first argument.

Like:
```python
    def update_elements(self, element_type: _pypowsybl.ElementType, df: _DataFrame = None, **kwargs):
        if df is None:
            ... transform kwargs in dataframe

```

**Comment by sylvlecl** [comment]
Implemented in #251
